test/harnesses/EverlongTest.sol

```diff
@@ -37,51 +57,92 @@ contract EverlongTest is HyperdriveTest, IEverlongEvents {
     // IHyperdriveCheckpointRewarder checkpointRewarder
     // IHyperdrive                   hyperdrive
 
-    /// @dev Everlong instance to test.
-    EverlongExposed internal everlong;
+    // ╭───────────────────────────────────────────────────────────────────────╮
+    // │                       Hyperdrive Configuration                        │
+    // ╰───────────────────────────────────────────────────────────────────────╯
+
+    address internal HYPERDRIVE_INITIALIZER = address(0);
+
+    uint256 internal FIXED_RATE = 0.05e18;
+    int256 internal VARIABLE_RATE = 0.05e18;
+
+    uint256 internal INITIAL_VAULT_SHARE_PRICE = 1e18;
+    uint256 internal INITIAL_CONTRIBUTION = 1_000_000e18;
+
+    uint256 internal CURVE_FEE = 0.01e18;
+    uint256 internal FLAT_FEE = 0.0005e18;
+    uint256 internal GOVERNANCE_LP_FEE = 0.15e18;
+    uint256 internal GOVERNANCE_ZOMBIE_FEE = 0.03e18;
```

Comment:
Nit: Isn't this missing natspec? I thought we were going to document harnesses more thoroughly. Let me know if I'm forgetting a conversation.